### PR TITLE
Note regression of CLI for dotnet run "-p" option

### PR DESCRIPTION
**Pull request by guardrex**
RE: https://github.com/dotnet/cli/issues/5992

Thanks to @dasmulli :+1: for the tip-off on this one.

### CODE REVIEW

File: docs/core/tools/dotnet-run.md [hunk 1]
```diff
@@ -60,7 +60,10 @@ Builds and runs the app using the specified [framework](../../standard/framework
 
 `-p|--project <PATH>`
 
-Specifies the path to the project file. It can be a path to a [csproj](csproj.md) file or to a directory containing a [csproj](csproj.md) file. It defaults to current directory if not specified. 
+Specifies the path to the project file. It can be a path to a [csproj](csproj.md) file ~or to a directory containing a [csproj](csproj.md) file~ (See NOTE). It defaults to current directory if not specified.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-run.md]: Removing the or clause makes the "can" inaccurate. I'd change this to "Specifies the path and name of the project file. (See the NOTE.)"
to current directory --> to the current directory

File: docs/core/tools/dotnet-run.md [hunk 2]
```diff
@@ -60,7 +60,10 @@ Builds and runs the app using the specified [framework](../../standard/framework
...
+
+> [!NOTE]
+> Use the full path to the project file with the `-p|--project` option. A regression in the CLI prevents providing a folder path at this time. For more information and to track this issue, see [dotnet run -p, can not start a project (dotnet/cli #5992)](https://github.com/dotnet/cli/issues/5992).
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-run.md]: The full path implies that relative paths aren't supported. I'd change this to "Use the path and name of the project file..."

**Review by rpetrusha — CHANGES_REQUESTED** [review]

File: docs/core/tools/dotnet-run.md (hunk 1 above)
Comment (GuardRex) [inline docs/core/tools/dotnet-run.md]: It's drawn thu ~or to a directory containing a [csproj](csproj.md) file~ ... but let's remove it as you say.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
LGTM. This can be merged once the build finishes.

**Comment by GuardRex** [comment]
Thanks @rpetrusha ... I'll track and revert this when they get it fixed.

**Pull request merged**